examples/shader-toy/main.cpp

```diff
@@ -82,7 +85,7 @@ void diagnoseIfNeeded(slang::IBlob* diagnosticsBlob)
 {
     if( diagnosticsBlob != nullptr )
     {
-        reportError("%s", (const char*) diagnosticsBlob->getBufferPointer());
+        printf("%s", (const char*) diagnosticsBlob->getBufferPointer());
     }
```

Comment:
What is the rationale for removing this as a bit of utility code that applications can use? I realize that `reportError` isn't a full-fledged logging system, but I would kind of expect us to move in that direction (making our logging more centralized and feature-full) rather than in the opposite direction.

I feel like I'm missing part of the rationale for this PR.